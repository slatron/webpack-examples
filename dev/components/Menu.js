import React, { useState } from 'react'

const Menu = (props) => {
  return (
    <ul>
      <li><a href="/index.html">Home</a></li>
      <li><a href="/contact.html">Contact</a></li>
    </ul>
  )
}

export default Menu
