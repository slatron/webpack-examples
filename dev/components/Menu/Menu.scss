// unique styles for the Menu.js component
ul.site-nav {
  li {
    display: inline-block;
    margin-right: 1em;
  }

  li.active {
    font-size: 28px;
  }
}
