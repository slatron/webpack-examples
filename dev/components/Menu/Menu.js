import React, { useState } from 'react'
import './Menu.scss'

const Menu = (props) => {
  return (
    <ul className="site-nav">
      <li><a href="/index.html">Home</a></li>
      <li><a href="/contact.html">Contact</a></li>
    </ul>
  )
}

export default Menu
