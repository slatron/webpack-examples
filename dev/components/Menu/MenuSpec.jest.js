import React from 'react';
import renderer from 'react-test-renderer';
import Menu from './Menu';

describe('Menu Display Test > ', () => {
  test('Renders Active Class Passed as current', () => {
    const MenuComponent = renderer.create(<Menu current="contact" />);
    let tree = MenuComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
