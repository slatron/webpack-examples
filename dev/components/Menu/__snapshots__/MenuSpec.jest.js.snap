// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Display Test >  Renders Active Class Passed as current 1`] = `
<ul
  className="site-nav"
>
  <li
    className=""
  >
    <a
      href="/newpage.html"
    >
      Home
    </a>
  </li>
  <li
    className="active"
  >
    <a
      href="/contact.html"
    >
      Contact
    </a>
  </li>
</ul>
`;
